session04/mailroomDictFile.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+"""
+Making changes to original mailroom assignment.
+Changes are:
+changing list of donor names from a list to a dictionary,
+adding new menu option (Send letters to everyone),
+printing letters for new and existing donors,
+and creating a txt file for each letter with donor's name as file's name
+"""
+
+# creating global list so it can be accessed from different functions
+# donor_list is not a diction
+donor_dict = {'Albert White': [1254.45, 8966.70],
+              'Susan Smith': [9856.94, 1245.78],
+              'Daniel Motoko': [5897.85, 20000.48],
+              'Alison Dempsey': [2565.44, 500.32],
+              'Patrick Johnson': [6578.55, 4879.23, 5879.99]}
+
+
+def menu():
+    """ print menu for user to select from """
+    question = input("\nPlease select options 1, 2, or 3, 4 from the menu "
+                         "or 'list' to view donor names: \n"
+                         "1. Send a Thank you letter\n"
+                         "2. Create a Report\n"
+                         "3. Send letters to everyone\n"
+                         "4. Quit\n"
+                         "What's your choice?\n")
+    return question
+
+def print_donor_list():
+    """ printing 0th element from dict (printing only donors' name)"""
+    print("Donor Name:\n")
+    for name in donor_dict:
+        print(name)
+
+def find_donor(name):
+    """ finding donor's name in donor_dict and printing their information """
```

Comment:
Technically speaking, you are returning the donor's name, rather than printing anything here. Always a good idea to update your docstrings when you change your code. :)